app/core/src/test/java/com/fsck/k9/cache/ExpiringCacheTest.kt

```diff
@@ -57,7 +59,7 @@ class ExpiringCacheTest {
     }
 
     private fun advanceClockBy(timeInMillis: Long) {
```

Comment:
It feels like this will be useful in a lot of tests. Maybe this should be a `TestClock` method?

Reply:
I changed the `TestClock` to allow advancetTimeBy duration